test/Dapr.AspNetCore.IntegrationTest.App/DaprController.cs

```diff
@@ -22,6 +22,12 @@ public void TopicB()
         {
         }
 
+        [CustomTopic("pubsub", "C")]
```

Comment:
LGTM overall, Could you also add an example showcasing the usefulness of this when reading from env var as the usecase as explained in the issue

Reply:
Sure, can do. Do you want it as a Unit Test or should it be added to one of the example projects? E.g. here:
https://github.com/dapr/dotnet-sdk/blob/master/examples/AspNetCore/ControllerSample/Controllers/SampleController.cs

Reply:
> Sure, can do. Do you want it as a Unit Test or should it be added to one of the example projects? E.g. here:
> https://github.com/dapr/dotnet-sdk/blob/master/examples/AspNetCore/ControllerSample/Controllers/SampleController.cs

adding it to one of the example project would be good.

Reply:
Added example in commit 09440a68f06ba02b982978c8de9b6b734aa8d482